pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -185,40 +185,42 @@ void processArguments() throws Exception {
      */
     @Getter
     abstract class FunctionDetailsCommand extends BaseCommand {
-        @Parameter(names = "--fqfn", description = "The Fully Qualified Function Name (FQFN) for the function")
+        @Parameter(names = "--fqfn", description = "The Fully Qualified Function Name (FQFN) for the function"
+                + " #Java,Python")
         protected String fqfn;
-        @Parameter(names = "--tenant", description = "The tenant of a Pulsar Function")
+        @Parameter(names = "--tenant", description = "The tenant of a Pulsar Function #Java,Python,Go")
```

Comment:
```suggestion
        @Parameter(names = "--tenant", description = "The tenant of a Pulsar Function #Java, Python, Go")
```
Can you add a space after `comma`? That's the basic English rule 😀

Screeshots LGTM

Reply:
Okay, I've added the space now.